### ISSUE TITLE
Eslint error on json file

### ISSUE THREAD

**Issue by Piterden** [CompuIves/codesandbox-client#327]
I have an error which, I believe shouldn't be there.
![image](https://user-images.githubusercontent.com/5930429/33234256-9ddb440c-d234-11e7-9b27-7775e628e5c8.png)

**Comment by Piterden** [comment]
Why don't me have a `.eslintrc` in newly created github repo?

**Comment by Piterden** [comment]
Haha! But in `.vue` file eslint not works ))

![image](https://user-images.githubusercontent.com/5930429/33234303-66bdc2d2-d235-11e7-872a-2ae0b4bf5d41.png)

**Comment by CompuIves** [comment]
Ahh, this is a good one @Piterden. The problem currently is in this check:

https://github.com/CompuIves/codesandbox-client/blob/master/packages/app/src/app/components/sandbox/CodeEditor/Monaco.js#L556-L557

We should make this check for `.js`, `.jsx`, `.ts` and `.tsx` files. Would you be interested in opening a PR for this?

> Why don't me have a `.eslintrc` in newly created github repo?

This is on the planning. But there are still some problems with plugin resolving/downloading before we can add this.

> Haha! But in .vue file eslint not works ))

That's because Vue isn't completely a `js` file. Vetur fixes this by enabling eslint for a specific range I believe, we can add this later on.

**Comment by Piterden** [comment]
I use codemirror

**Comment by CompuIves** [comment]
Right, for codemirror we need to change the `getMode` a bit (https://github.com/CompuIves/codesandbox-client/blob/master/packages/app/src/app/components/sandbox/CodeEditor/CodeMirror.js#L321-L322). We should add `.(t|j)sx?` detection manually and fallback to `text` type.

**Comment by Piterden** [comment]
Why don't you write comments on functions?
Like here https://github.com/Piterden/my_music_bot/blob/1.0/src/TgKeyboard.js

**Comment by CompuIves** [comment]
Yes, I should definitely add more comments. The newer code has more inline documentation (because that was meant to be open sourced).

If you'd like to write more documentation, feel free to!

**Comment by Piterden** [comment]
I want, but I need to dig into it a bit, at the first...
